### ISSUE TITLE
Start the path after the filename

### ISSUE THREAD

**Issue by agaro1121** [loctvl842/winbar#1]
Is there any way to start the breadcrumbs at the file name?  The full breadcrumbs can get pretty lengthy on a small monitor like  ie 14" laptop

<img width="1009" alt="image" src="https://user-images.githubusercontent.com/2607903/235266478-5cca4c9e-7e62-42d2-ad7f-91ca7285be80.png">

In the screenshot above, I'd like to truncate everything before `breadcrumb.lua`

**Comment by loctvl842** [comment]
yes, you can file the solution in function get_filepath in file `renderer.lua`. Then just you the `cur_filename`. Remove anything unnecessary
